tests/regex/s2n_regex_test.py

```diff
@@ -0,0 +1,12 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
1. can we get rid of all of the `__init__.py` files?
2. having a test is nice, but I'm not sure the extra headache is worth it. generally we want our helper scripts to be simple enough that separate unit tests aren't necessary. If you feel that things are getting too complicated, your first preference should be to try and make the code simple enough that it doesn't need a test. Adding unit tests to helper scripts is a matter of last resort 😄